### ISSUE TITLE
Feature: Quick Navigation/Search

### ISSUE THREAD

**Issue by SlowLogicBoy** [0xd4d/dnSpy#271]
Ctrl + Q / Ctrl + N / Ctrl + T to open dialog like this:
![Go To Everything](https://www.jetbrains.com/resharper/features/screenshots/100/go_to_everything.png)
Search and Navigate like in Search tab.

**Comment by pardeike** [comment]
The general searchability is sub-par right now. This Issue would fix a lot but I see that you have removed priority from it. Seems fair. Keep in mind though that the current version returns a million constructors if you seach for a method: [https://pasteboard.co/IpBAh8e.png](url)

**Comment by 0xd4d** [comment]
All labels were removed, this issue is still open. Try 'match whole words'

**Comment by pardeike** [comment]
Nope, I tried all combinations of search parameters. I have to sort of type a full regex to get around this - not something I want to do 500 times a day. I might fix this too if you don't have time.
